pipelines/skylab/multiome/atac.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.8
```

Comment:
@aawdeh would you mind updating the version number of the ATAC pipeline in the README before you merge? Thanks!

Reply:
Done!